applications/app/templates/serviceWorker.scala.js

```diff
@@ -92,63 +92,3 @@ this.addEventListener('fetch', function (event) {
         event.respondWith(forbidden);
     }
 });
-
-self.addEventListener('push', function (event) {
```

Comment:
I do wonder what useful functionality we want to keep in our service worker. I can see the IAS exclusion, which we may not need anymore, and the credentials work around, which I'm also not sure we need anymore.